docs/framework/docker/aspnetmvc.md

```diff
@@ -50,33 +45,31 @@ The finished application is located in the
 
 ## Prerequisites
 
-At a minimum, your development machine must be running
+The development machine must be running
```

Comment:
if you remover the at a minimum, then I think you should add or later versions to the end, no?